src/client/mod.rs

```diff
@@ -129,7 +130,15 @@ impl Client {
 
     /// Returns a list of topics in the cluster
     pub async fn list_topics(&self) -> Result<Vec<Topic>> {
```

Comment:
`list_topics()` will always refresh the cached entry as a "free" side effect, as will creating a topic, or an explicit call to `refresh_metadata()`